tensorflow_addons/metrics/cohens_kappa.py

```diff
@@ -67,18 +67,26 @@ def __init__(self,
                  num_classes: FloatTensorLike,
                  name: str = 'cohen_kappa',
                  weightage: Optional[str] = None,
+                 sparse_labels: bool = False,
+                 reg_values: bool = False,
                  dtype: AcceptableDTypes = None,
                  **kwargs):
         """Creates a `CohenKappa` instance.
 
         Args:
           num_classes: Number of unique classes in your dataset.
-          name: (Optional) String name of the metric instance.
-          weightage: (Optional) Weighting to be considered for calculating
+          weightage: (optional) Weighting to be considered for calculating
             kappa statistics. A valid value is one of
-            [None, 'linear', 'quadratic']. Defaults to `None`.
-          dtype: (Optional) Data type of the metric result.
-            Defaults to `None`.
+            [None, 'linear', 'quadratic']. Defaults to `None`
+          sparse_lables: (bool) Valid only for multi-class scenario.
+            If True, ground truth labels are expected tp be integers
+            and not one-hot encoded
+          reg_values: (bool) If set, that means the problem is being treated
```

Comment:
nit: Could we make this `regression (bool)` . `reg_values` seems a bit underspecified since `reg` could mean regression, regularization, etc. Granted there wouldn't be regularization in this metric but its in the vocabulary.

Reply:
Makes sense